metainsuranceorg.py

```diff
@@ -433,3 +442,34 @@ def process_newrisks_insurer(self, risks_per_categ, number_risks_categ, acceptab
 
         return not_accepted_risks
 
+
+    def market_permanency(self, time):     #This method determines whether an insurer or reinsurer stays in the market. If it has very few risks underwritten or too much cash left for TOO LONG it eventually leaves the market.
```

Comment:
We should make it possible to switch this off. Or better yet include it in the calibration with some kind of `market_permanence_intensity` parameter. But forcing this would be non-generic (and would also duplicate the illiquidity method and mechanism).

Reply:
Done.